README.md

```diff
@@ -13,6 +13,16 @@ designing government services.**
 
 Make sure you have [Node.js](https://nodejs.org/en/) installed.
 
+Note: You will need the [active LTS (Long-term support)](https://github.com/nodejs/Release#release-schedule) Node.js version for this project (as specified in [.nvmrc](./.nvmrc))
```

Comment:
In which case do we want `.npmrc` to reference `lts/*`?

Reply:
as a reference, we use `lts/*` in prototype kit travis script
https://github.com/alphagov/govuk_prototype_kit/blob/master/.travis.yml#L3

Reply:
I'd prefer we pin a version so we don't have to re-download Node.js whenever a new release comes along, and then update this pinned version as and when.

Reply:
But we're asking contributors to download a specific version of npm when they might already have an `8.x` they can use (perhaps even a newer one)? If nvm sees lts/* will it not just use _any_ locally installed version that satisfies that condition?

Reply:
I imagine it should work that way but this PR is to avoid people installing a version that we don't use that could get in the way of them contributing.

If someone chooses not to use nvm, or only wants to use a major release, worse case they get a warning that they can choose to action or not.